### ISSUE TITLE
provenance check is not producing documents, even though requests are coming in.

### ISSUE THREAD

**Issue by pacospace** [thoth-station/thoth-application#1212]
**Describe the bug**

**To Reproduce**
1. Schedule provenance checke

**Expected behavior**
documents are produced by the service once a request has been created

**Screenshots**
```
INFO:thoth.thoth_reporter:Analyzing for start date: 2021-04-22
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-04-23
INFO:thoth.reporter.processing:components requests stats:
date`component`requests`reports`
2021-04-22`adviser`12`12
2021-04-22`package-extract`0`0
2021-04-22`provenance-checker`0`0

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-04-23
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-04-24
INFO:thoth.reporter.processing:components requests stats:
date`component`requests`reports`
2021-04-23`adviser`12`12
2021-04-23`package-extract`0`0
2021-04-23`provenance-checker`1`0

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-04-24
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-04-25
INFO:thoth.reporter.processing:components requests stats:
date`component`requests`reports`
2021-04-24`adviser`17`17
2021-04-24`package-extract`0`0
2021-04-24`provenance-checker`7`0

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-04-25
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-04-26
INFO:thoth.reporter.processing:components requests stats:
date`component`requests`reports`
2021-04-25`adviser`17`16
2021-04-25`package-extract`0`0
2021-04-25`provenance-checker`6`0

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-04-26
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-04-27
INFO:thoth.reporter.processing:components requests stats:
date`component`requests`reports`
2021-04-26`adviser`18`17
2021-04-26`package-extract`0`0
2021-04-26`provenance-checker`5`0

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-04-27
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-04-28
INFO:thoth.reporter.processing:components requests stats:
date`component`requests`reports`
2021-04-27`adviser`13`12
2021-04-27`package-extract`0`0
2021-04-27`provenance-checker`4`0

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-04-28
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-04-29
INFO:thoth.reporter.processing:components requests stats:
date`component`requests`reports`
2021-04-28`adviser`14`14
2021-04-28`package-extract`0`0
2021-04-28`provenance-checker`3`0
```

**Additional context**

**Comment by goern** [comment]
Which deployment is this? is there an integration-test related?

/label triage/needs-information

**Comment by harshad16** [comment]
This issue was worked on and fixed: https://github.com/thoth-station/thoth-application/issues/1228
can we re-check on this one.

**Comment by pacospace** [comment]
> This issue was worked on and fixed: #1228
> can we re-check on this one.

```
INFO:thoth.thoth_reporter:Analyzing for start date: 2021-05-04
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-05-05
INFO:thoth.reporter.processing:components requests stats:
2021-05-04`adviser`23`9
2021-05-04`package-extract`0`0
2021-05-04`provenance-checker`10`2

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-05-05
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-05-06
INFO:thoth.reporter.processing:components requests stats:
2021-05-05`adviser`64`62
2021-05-05`package-extract`0`0
2021-05-05`provenance-checker`9`9

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-05-06
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-05-07
INFO:thoth.reporter.processing:components requests stats:
2021-05-06`adviser`28`27
2021-05-06`package-extract`0`0
2021-05-06`provenance-checker`5`5

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-05-07
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-05-08
INFO:thoth.reporter.processing:components requests stats:
2021-05-07`adviser`42`42
2021-05-07`package-extract`0`0
2021-05-07`provenance-checker`7`7

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-05-08
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-05-09
INFO:thoth.reporter.processing:components requests stats:
2021-05-08`adviser`22`21
2021-05-08`package-extract`0`0
2021-05-08`provenance-checker`2`1

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-05-09
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-05-10
INFO:thoth.reporter.processing:components requests stats:
2021-05-09`adviser`20`19
2021-05-09`package-extract`0`0
2021-05-09`provenance-checker`3`0

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-05-10
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-05-11
INFO:thoth.reporter.processing:components requests stats:
2021-05-10`adviser`53`49
2021-05-10`package-extract`0`0
2021-05-10`provenance-checker`1`1

INFO:thoth.thoth_reporter:Analyzing for start date: 2021-05-11
INFO:thoth.thoth_reporter:Analyzing for end date (excluded): 2021-05-12
INFO:thoth.reporter.processing:components requests stats:
2021-05-11`adviser`43`42
2021-05-11`package-extract`0`0
2021-05-11`provenance-checker`2`2
```

**Comment by pacospace** [comment]
We can close the issue. It is working correctly now from data prospective, we added a new alarm to be set: https://github.com/thoth-station/thoth-application/issues/854
